src/applications/vaos/components/NoValidVAFacilities.jsx

```diff
@@ -1,28 +1,88 @@
 import React from 'react';
-import { Link } from 'react-router';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import LocationDirectionsLink from 'applications/facility-locator/components/search-results/LocationDirectionsLink';
```

Comment:
We could ask VSP about this, but I don't think we want to rely on code in other applications. It'd be easy for someone working on FL to not realize we're using this component.